tests/categories.test.ts

```diff
@@ -0,0 +1,26 @@
+import { test } from "uvu";
+import * as assert from "uvu/assert";
+import { categories } from "../src";
+
+test("are properly sorted", () => {
+  const keys = Object.keys(categories);
+  const sortedKeys = Object.keys(categories).sort();
+
+  assert.is(keys.join("\n"), sortedKeys.join("\n"));
+});
+
+test("have valid category keys", () => {
+  const keys = Object.keys(categories);
+
+  assert.ok(keys.join("").match(/^[a-z0-9-]+$/)); // lowercase letters, numbers and dashed only
+});
+
+test("have valid number of suggestions", () => {
+  const MAX_SIZE = 7;
```

Comment:
@pastelsky Some of the categories have 6-7 packages in "similar" field, so I had to set this variable to 7